src/legacy/templates/workEditT16.handlebars

```diff
@@ -82,6 +82,10 @@
                         <label for="cancelled">Cancelled </label>
                         <input type="checkbox" name="cancelled" value="false" class="checkbox"/>
                     </div>
+                    <div id="survey">
+                        <label for="survey-checkbox">Census </label>
+                        <input id="survey-checkbox" type="checkbox" name="survey" {{#if this.description.survey}}checked{{/if}}/>
```

Comment:
A bit picky here but the naming is not very accurate. This is the census checkbox. But we also talked about this potentially evolving to a survey dropdown and made the backend change accordingly (hence the name of the field being survey)

My view is that this should be just about census for now to avoid any confusion. If we were to add surveys in the future, this would need to change anyway as there won't be a survey checkbox so better name it accurately now?

Reply:
Fair